src/python/ASO/Rucio/RunTransfer.py

```diff
@@ -33,14 +34,23 @@ def algorithm(self):
         self.transfer = Transfer()
         self.transfer.readInfo()
         self.rucioClient = self._initRucioClient(self.transfer.username, self.transfer.restProxyFile)
-        #self.crabRESTClient = self._initCrabRESTClient
-        # do dbs dataset
-        #BuildDBSDataset(self.transfer, self.rucioClient).execute()
+        self.transfer.readInfoFromRucio(self.rucioClient)
```

Comment:
readInfoFromRucio is not a good name. Name should indicate the action, not the means used to perform it. Maybe something like populateLfn2DatasetMap ?